### PR TITLE
NR-30929: Ansible-Tower-Integration

### PR DESCRIPTION
**Pull request by pkosanam**
# Summary

Ticket URL: https://issues.newrelic.com/browse/NR-30929

Description:

This PR has the new quickstart integration named "Ansible tower"

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.
ansible-tower-preview

### CODE REVIEW

File: quickstarts/ansible-tower/config.yml
Comment (aswanson-nr) [inline quickstarts/ansible-tower/config.yml:63]: This is most likely where that failure is coming from, this just needs to be the directory your dashboard is in.
```suggestion
  - ansible-tower
```

**Review by aswanson-nr — COMMENTED** [review]
I think I found the spot causing that error @pkosanam

File: quickstarts/ansible-tower/config.yml
Comment (pkosanam) [reply quickstarts/ansible-tower/config.yml]: Thank you @aswanson-nr . Made required changes 👍

File: dashboards/ansible-tower/ansible-tower.json
Comment (aswanson-nr) [inline dashboards/ansible-tower/ansible-tower.json:1]: I noticed some entity guids and account ids in here, let's get those cleaned up before merge. The `sanitize-dashboard` utility can automatically do that for you, docs are [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards)

File: quickstarts/ansible-tower/config.yml
Comment (aswanson-nr) [inline quickstarts/ansible-tower/config.yml:1]: This is be automatically added after merging!
```suggestion
```

File: dashboards/ansible-tower/ansible-tower.json
Comment (pkosanam) [reply dashboards/ansible-tower/ansible-tower.json:1]: awesome.. thank you @aswanson-nr . Dashboard is now sanitised. 😃

**Comment by pkosanam** [comment]
Here is the preview image:
screencapture-newrelic-instant-observability-preview-2022-07-29-08_37_52

**Comment by pkosanam** [comment]
Please do not merge it as we are working on docs link which has to be integrated in this PR 🙂

**Comment by aswanson-nr** [comment]
Hey @pkosanam! I'm going to move this into draft so the team knows not to merge it. Just toss it back to "Ready for review" when y'all are ready for a merge!

**Comment by aswanson-nr** [comment]
Not stale!

File: dashboards/ansible-tower/ansible-tower.json
Comment (zstix) [inline dashboards/ansible-tower/ansible-tower.json:128]: It looks like the `entity.guid` is hard-coded in this query. We should remove this to ensure that it works the same for all users that install it.

File: dashboards/ansible-tower/ansible-tower.json
Comment (zstix) [inline dashboards/ansible-tower/ansible-tower.json:578]: There is another reference to a specific `entityGuid` in this query. We should remove this.

File: dashboards/ansible-tower/ansible-tower.json
Comment (zstix) [inline dashboards/ansible-tower/ansible-tower.json:572]: We probably want to set this to `0` so that all users will see the same experience.

```suggestion
                  "accountId": 0,
```

File: dashboards/ansible-tower/ansible-tower.json
Comment (zstix) [inline dashboards/ansible-tower/ansible-tower.json:576]: What is this used for? Will all users have the same `agent_ids`?

File: dashboards/ansible-tower/ansible-tower.json
Comment (zstix) [inline dashboards/ansible-tower/ansible-tower.json:617]: We should remove `entityGuid` here.

File: dashboards/ansible-tower/ansible-tower.json
Comment (zstix) [inline dashboards/ansible-tower/ansible-tower.json:656]: We should remove `entityGuid` here.

File: dashboards/ansible-tower/ansible-tower.json
Comment (zstix) [inline dashboards/ansible-tower/ansible-tower.json:683]: This `entityGuid` is hard-coded. If we're unable to display logs without a specific entity, perhaps we should remove this widget.

File: install/ansible-tower/install.yml
Comment (zstix) [inline install/ansible-tower/install.yml:13]: We should provide a better documentation source for installing this setup. Perhaps we can use the [ansible docs](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/config-management-tools/configure-infrastructure-agent-using-ansible/). I'd suggest reaching out to someone on the docs team.

**Review by zstix — CHANGES_REQUESTED** [review]
There are a few things that need to be cleaned up from the dashboard file. There are a number of instances where we have the `entityGuid` hard-coded. I'd suggest running the dashboard cleanup script to remove those.

**Comment by pkosanam** [comment]
1. Updated dashboard and alert policies
2. Updated Preview image
To do:
1. Update docs urls by coordinating with docs team

File: dashboards/ansible-tower/ansible-tower.json
Comment (pkosanam) [reply dashboards/ansible-tower/ansible-tower.json]: done.. removed it 👍

File: dashboards/ansible-tower/ansible-tower.json
Comment (pkosanam) [reply dashboards/ansible-tower/ansible-tower.json]: done.. removed it and updated the dashboard json

File: dashboards/ansible-tower/ansible-tower.json
Comment (pkosanam) [reply dashboards/ansible-tower/ansible-tower.json]: done.. updated the dashboard json

File: dashboards/ansible-tower/ansible-tower.json
Comment (pkosanam) [reply dashboards/ansible-tower/ansible-tower.json]: done.. removed it and updated the dashboard json

File: dashboards/ansible-tower/ansible-tower.json
Comment (pkosanam) [reply dashboards/ansible-tower/ansible-tower.json]: done.. removed it and updated the dashboard json

File: dashboards/ansible-tower/ansible-tower.json
Comment (pkosanam) [reply dashboards/ansible-tower/ansible-tower.json]: done.. removed it and updated the dashboard json

File: dashboards/ansible-tower/ansible-tower.json
Comment (pkosanam) [reply dashboards/ansible-tower/ansible-tower.json]: done.. removed it and updated the dashboard json

File: install/ansible-tower/install.yml
Comment (pkosanam) [reply install/ansible-tower/install.yml:13]: on hold, in coordination with @akristen on it.

Comment (pkosanam) [reply install/ansible-tower/install.yml:13]: Hi @akristen, request you to kindly provide us the docs url, so that we can update this PR with it. 😃

File: quickstarts/ansible-tower/config.yml
Comment (aswanson-nr) [inline quickstarts/ansible-tower/config.yml:51]: Looks like we just need to get this url and the one in the install plan up to date and we should be good to go!

**Review by aswanson-nr — COMMENTED** [review]
Looks good from my perspective once we get those documentation links updated!

**Pull request closed without merging**
